### PR TITLE
Update paypal_invoice_abuse.yml

### PR DESCRIPTION
**Pull request by morriscode**
# Description
Extending phone number patterns, also adding optional considerations for the word Kindly. 
Updated the paypal template to accomidate "Notes from X" in addition to "Seller note"

# Associated samples

- [Sample 1](https://platform.sublime.security/rules/editor?canonical_id=54dcf8a37463a09782ae6d10315f5e2ba236aecbf2451a602a7b3c9bdd3116de)

### CODE REVIEW

**Comment by morriscode** [comment]
Thinking about it, looking for the phone number in the comment section is almost conclusive enough without the additional keyword context matching.

File: detection-rules/paypal_invoice_abuse.yml [hunk 1]
```diff
@@ -12,36 +12,82 @@ source: |
...
-                     '[\s:,-]\+?\d{1,2}[\s:,-]\(?([2-7][0-9]{2}|80[1-9]|8[1-9][0-9])\)?[\s:,-]\d{3}[\s:,-]\d{4}\b'
+      // icontains a phone number
+      (
+        regex.icontains(strings.replace_confusables(body.current_thread.text),
+                        'note from.{0,}\+?(\d{1}.)?\(?\d{3}?\)?.\d{3}.?\d{4}.{0,}Transaction ID'
+        )
+        or regex.icontains(strings.replace_confusables(body.current_thread.text),
+                           'note from.{0,}\+\d{1,3}[0-9]{10}.{0,}Transaction ID'
+        )
+        or // +12028001238
```
Comment (zoomequipd) [inline detection-rules/paypal_invoice_abuse.yml:29]: I thought we fixed these comments?

**Comment by zoomequipd** [comment]
added commit to hit on an additional sample and correct phone number formats in comments

[Sample](https://platform.sublime.security/messages/beb94a2d853fe12710b91144fadbaeaf63e2a8bfa74bfc539d9ed85328a54487?ph-e6489411-a22d-4959-9ecc-30170413281d=019002b4-05ed-7bf5-8e38-692645fd5155)

File: detection-rules/paypal_invoice_abuse.yml (hunk 1 above)
Comment (morriscode) [reply detection-rules/paypal_invoice_abuse.yml]: hmm I thought we did too.

Comment (morriscode) [reply detection-rules/paypal_invoice_abuse.yml:29]: Just modified it again, was this where we landed?

**Pull request closed without merging**